### PR TITLE
jom: Download binaries in build method and fix test packages

### PR DESCRIPTION
**Pull request by jwillikers**
Use conan.tools.files.copy.
Update license to be GPL-3.0-only.

Specify library name and version:  **lib/1.0**

This is also a good place to share with all of us **why you are submitting this PR** (specially if it is a new addition to ConanCenter): is it a dependency of other libraries you want to package? Are you the author of the library? Thanks!

---

- [ ] I've read the [contributing guidelines](https://github.com/conan-io/conan-center-index/blob/master/CONTRIBUTING.md).
- [ ] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [ ] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [ ] I've tried at least one configuration locally with the [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Comment by uilianries** [comment]
Just restarted it now.

**Review by uilianries — APPROVED** [review]
LGTM

File: recipes/jom/all/conanfile.py [hunk 1]
```diff
@@ -43,8 +44,8 @@ def package_info(self):
 
         bin_folder = os.path.join(self.package_folder, "bin")
         # In case need to find packaged tools when building a package
-        self.buildenv_info.append("PATH", bin_folder)
+        self.buildenv_info.prepend_path("PATH", bin_folder)
```
[the remark this answers is not in the document]
Comment (jwillikers) [reply recipes/jom/all/conanfile.py]: Thanks. That's what I thought, but wasn't sure why these had been added initially. I've removed these statements.

**Review by uilianries — APPROVED** [review]
LGTM

**Comment by jwillikers** [comment]
@ericLemanissier PTAL, thanks!

**Review by toge — APPROVED** [review]

**Pull request merged**